### ISSUE TITLE
Twitter login getting email address of the user

### ISSUE THREAD

**Issue by laziz-karimov** [yiisoft/yii2-authclient#62]
Hi everyone!

I could successfully get emails of the user while logging in with google, facebook, and vkontakte.
Twitter by default doesno allow getting an email of the user. 
After a request I got this permission and in my Twitter apps I have 
<b>Request email addresses from users</b> as checked.

But in the result array I get a lot of data with twits, username, timezone, etc. except the email.
What can be the problem?

**Comment by samdark** [comment]
Probably you should specify that you need email somehow. Check how to do it in twitter API docs.

**Comment by laziz-karimov** [comment]
Thank you for response.

Unfortunately I found nothing in the twitter documentation. The only thing they say developers must separately ask the Twitter team to allow your app send the email. From the twitter side everything looks fine.

As I did not get the email for vk login I had to add this parameter to the request scope. I was thinking there is a similiar solution for Twitter. Adding email to the scope did not help.
